appveyor.yml

```diff
@@ -0,0 +1,96 @@
+-
+  branches:
+    only:
```

Comment:
If this can be configured to only build on tags that'd be great. These branches were a work around due to a weird timing bug with the node-sass appveyor setup. Turned out we were still firing a webhook to your personal node-sass appveyor. That was causing issues with our webhooks timing. Removing that made the `release` branch unnecessary.